src/mesido/workflows/utils/adapt_profiles.py

```diff
@@ -76,97 +75,163 @@ def adapt_hourly_year_profile_to_day_averaged_with_hourly_peak_day(problem, prob
+            except KeyError:
+                break
             set_data_with_averages_and_peak_day(
                 datastore=new_datastore,
                 variable_name=var_name,
                 ensemble_member=ensemble_member,
                 new_date_times=new_date_times,
                 problem=problem,
             )
-        # ------------------------------------------------------------------------------------------
-        # cooling demands
-        total_cold_demand = None
-        cold_demand_nominal = dict()
 
         for demand in cold_demands:
             try:
-                cold_demand_values = problem.get_timeseries(
-                    f"{demand}.target_cold_demand", ensemble_member
-                ).values
+                var_name = f"{demand}.target_cold_demand"
             except KeyError:
-                continue
-            if total_cold_demand is None:
-                total_cold_demand = cold_demand_values
-            else:
-                total_cold_demand += cold_demand_values
-            cold_demand_nominal[f"{demand}.Cold_demand"] = max(cold_demand_values)
-            cold_demand_nominal[f"{demand}.Heat_flow"] = max(cold_demand_values)
-
-        # TODO: find the peak cooling day and adapt to hourly
-
-        for demand in cold_demands:
-            var_name = f"{demand}.target_cold_demand"
+                break
             set_data_with_averages_and_peak_day(
                 datastore=new_datastore,
                 variable_name=var_name,
                 ensemble_member=ensemble_member,
                 new_date_times=new_date_times,
                 problem=problem,
             )
+        # ------------------------------------------------------------------------------------------
+        # cooling demands
+        # total_cold_demand = None
+        # cold_demand_nominal = dict()
+
+        # for demand in cold_demands:
+        #     try:
+        #         cold_demand_values = problem.get_timeseries(
+        #             f"{demand}.target_cold_demand", ensemble_member
+        #         ).values
+        #     except KeyError:
+        #         continue
+        #     if total_cold_demand is None:
+        #         total_cold_demand = cold_demand_values
+        #     else:
+        #         total_cold_demand += cold_demand_values
+        #     cold_demand_nominal[f"{demand}.Cold_demand"] = max(cold_demand_values)
+        #     cold_demand_nominal[f"{demand}.Heat_flow"] = max(cold_demand_values)
+
+        # # TODO: find the peak cooling day and adapt to hourly
+
+        # for demand in cold_demands:
+        #     var_name = f"{demand}.target_cold_demand"
+        #     set_data_with_averages_and_peak_day(
+        #         datastore=new_datastore,
+        #         variable_name=var_name,
+        #         ensemble_member=ensemble_member,
+        #         new_date_times=new_date_times,
+        #         problem=problem,
+        #     )
```

Comment:
I think this can be removed?

Reply:
Same as with the other comment. I only had this hanging around during development, it will be removed in the final commit.